package/CHANGELOG

```diff
@@ -18,6 +18,7 @@ The rules for this file:
          ooprathamm, MeetB7, BFedder, v-parmar, MoSchaeffler, jbarnoud, jandom,
          xhgchen, jaclark5, DrDomenicoMarson, AHMED-salah00, schlaicha, SophiaRuan,
 	 g2707
```

Comment:
This could also do with some indentation... =)

```suggestion
         xhgchen, jaclark5, DrDomenicoMarson, AHMED-salah00, schlaicha, SophiaRuan,
	     g2707
```

Reply:
Not sure I see it, it's already indented? the previous line exceeds 83 chars so I'll fix that.